vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "DevMenu.h"
+#include "Modules/DevSettingsModule.h"
+#include "UI.Xaml.Controls.Primitives.h"
+#include "UI.Xaml.Controls.h"
+#include "UI.Xaml.Input.h"
+#include "UI.Xaml.Interop.h"
+#include "UI.Xaml.Markup.h"
+#include "Utils/Helpers.h"
+#include "IReactDispatcher.h"
+#include "winrt/Windows.UI.Core.h"
+#include "Views/KeyboardEventHandler.h"
+
+namespace Microsoft::ReactNative {
+
+React::ReactPropertyId<React::ReactNonAbiValue<std::shared_ptr<DevMenuManager>>> DevMenuManagerProperty() noexcept
+{
+  static React::ReactPropertyId<React::ReactNonAbiValue<std::shared_ptr<DevMenuManager>>> propId {
```

Comment:
Does cppwinrt have anything UnknownObject style for normal objects to implement IInspectable? Feels a bit out of the way to add an intrusive ref count with enable_shared_from_this just to box it into an IInspectable wrapper.